docs/DeveloperGuide.md

```diff
@@ -162,7 +162,7 @@ When the `Recommend` class is instantiated, the constructor calls onto the `Stor
 
 The `recommend` method in the class then chooses and adds different permutations or combinations of `StandardExercise` instances from the multiple different `StandardExerciseList` instances to load into a new instance of `StandardExerciseList`.
 
-<p align="center"><img src="images/RecommenderClassDiagram.png"></p>
+<p align="center"><img src="images/RecommendSequence.png"></p>
 <p align="center">Figure 6: <code>Recommender</code> class diagram</p>
```

Comment:
Change caption to sequence diagram

Reply:
Wrongly put diagram at wrong place, Changed already